frontend/rolecall/angular.json

```diff
@@ -29,7 +29,8 @@
             ],
             "styles": [
               "./node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css",
-              "src/styles.scss"
+              "src/styles.scss",
+              "src/std.scss"
```

Comment:
Revert the changes to this file and let's get https://github.com/googleinterns/role-call/pull/27 merged in.

Reply:
Done.